server/src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -282,9 +272,7 @@ public ContentAccessCertificate getCertificate(Consumer consumer)
             Date expiration = existing.getSerial().getExpiration();
             Date contentUpdate = owner.getLastContentUpdate();
 
-            if (expiration.before(now) || !contentUpdate.before(existing.getUpdated()) ||
-                oeca == null || !contentUpdate.before(oeca.getUpdated())) {
-
+            if (expiration.before(now) || !contentUpdate.before(existing.getUpdated())) {
```

Comment:
I thought that with this change, we could get rid of the `Owner.getLastContentUpdate` workaround?